src/react-components/ui-root.js

```diff
@@ -817,7 +817,11 @@ class UIRoot extends Component {
 
   renderEntryStartPanel = () => {
     const { hasAcceptedProfile, hasChangedName } = this.props.store.state.activity;
-    const promptForNameAndAvatarBeforeEntry = this.props.hubIsBound ? !hasAcceptedProfile : !hasChangedName;
+    //const isLockedDownDemo = configs.feature("is_locked_down_demo_room") === this.props.hub.hub_id;
+    const isLockedDownDemo = true;
```

Comment:
Reminder to remove this line and uncomment the above before merging :)

Reply:
Thank you for catching this!